lua/autorun/client/cl_hideholos.lua

```diff
@@ -0,0 +1,12 @@
+
+net.Receive( "update_hologram_visibility_preference", function( len )
+    cookie.Set( "cfc_hologram_visibility_preference", net.ReadBool() )
+end )
+
+net.Receive( "get_user_hologram_visibility_preference", function( len )
+    local chip = cookie.GetNumber( "cfc_hologram_visibility_preference", 0 ) ~= 0
+
+    net.Start( "get_user_hologram_visibility_preference" )
```

Comment:
Should these net messages bet the same here? I would imagine one of them should be `Get` when the server requests the data from the client, and then when the client updates it it should be `Set`. Then maybe as a response to the `Get` we could have a `Receive` or something.

Reply:
I was just saving some lines, as far as I know this shouldn't affect the final output... I like to think of it as the server sends a GET request and the user pings back the info. I'll leave it up to you as far as preference.